tictactoe.py

```diff
@@ -100,8 +62,8 @@ def game(board):
         if end_game == 0:
             printBoard(board)
             play_a_turn(player_one_name,theBoard, True)
-            if i > 1:
-                check_winner(player_one_name, player_two_name,board)
+            if i > 1: # duplicate code here as line 70
```

Comment:
line 65 and line 70 are duplicated code, can you combine into one line?

Reply:
the duplicates at line 65 and line 70 are required because we need to check for the winner via check_winner() after each player make their move